tests/test_datacite.py

```diff
@@ -1,22 +1,119 @@
-from transmogrifier.helpers import parse_xml_records
-from transmogrifier.sources.datacite import Datacite
-
+from pytest import raises
 
-# TODO: add tests for expected data issues:
-# missing required Datacite fields raises error
-# all fields transformed successfully
-# repeatable fields transformed successfully
+from transmogrifier.models import Contributor, Date, Identifier, Note, TimdexRecord
+from transmogrifier.sources.datacite import Datacite
 
 
-def test_jpal_fixtures():
-    input_records = parse_xml_records("tests/fixtures/datacite_jpal_records.xml")
+def test_jpal_fixtures(datacite_jpal_records):
     output_records = Datacite(
         "jpal",
         "https://dataverse.harvard.edu/dataset.xhtml?persistentId=",
         "Abdul Latif Jameel Poverty Action Lab Dataverse",
-        input_records,
+        datacite_jpal_records,
     )
     assert len(list(output_records)) == 38
 
 
-# TODO: add tests for generate_name_identifier_url expected cases
+def test_datacite_full_record(datacite_record, datacite_jpal_record_full):
```

Comment:
I would use the same language here (and in conftest.py) that we use in the models tests, e.g. `datacite_record_all_fields`, `datacite_record_timdex_required_fields`, etc. as I think it's more clear than full and minimal.